README.md

```diff
@@ -70,6 +70,8 @@ Unknown properties are ignored.
 > }
 > ```
 
+##Published events
```

Comment:
You ate a space after `##`

![image](https://user-images.githubusercontent.com/11961746/77765455-93e89d00-703e-11ea-8a47-d5dc4c431d89.png)